inst/@sym/coeffs.m

```diff
@@ -202,7 +248,7 @@
 %! assert (isequal (t, s))
 
 %!test
-%! % empty same as no specifying
+%! % empty same as not specifying
```

Comment:
@cbm755 Hi, I was just trying to run these tests on MATLAB R2016a and it seems that MATLAB gives different outputs if the second argument is empty vs when it's not there.  
Example:  
```matlab
>> [c, t] = coeffs(6*x*x + 27*y*x  + 36, {})
c =
6*x^2 + 27*y*x + 36
t =
1
>> [c, t] = coeffs(6*x*x + 27*y*x  + 36)
c =
[ 6, 27, 36]
t =
[ x^2, x*y, 1]
>> 
```
( I just reducing the spacing in the output above).  
As it seems that the order of `c` and `t` are reversed in this case. Also, MATLAB doesn't give any error/warning if the `all` is misspelled. It just treats it as giving `{}` as the argument. In all cases where there is an argument and it is not exactly `all` then it gives the same result. So, `[c, t] = coeffs(6*x*x + 27*y*x  + 36, {});` gives the same result as `[c, t] = coeffs(6*x*x + 27*y*x  + 36, 'some weird stuff');`